src/controllers/products.js

```diff
@@ -0,0 +1,13 @@
+import productsData from '../data/data';
+
+export const list = (req, res) => {
+  res.send(productsData);
+};
+
+export const detail = (req, res) => {
+  const { id } = req.params;
+  const detailData = productsData.data.filter(
+    (product) => product.id.toString() === id
+  );
```

Comment:
```suggestion
  const detailData = productsData.data.filter(
    (product) => product.id.to === Number(id)
  );
```
기능은 정확히 동일하게 동작하겠지만, 의미적으로 id가 숫자이다보니 이렇게 비교하는 것을 추천드려봅니다 :)
이는 참고 사항이라 꼭 수정하지는 않으셔도 됩니다.

다만, 파라미터를 비교할 때 이런 점도 고려하는 팀도 있습니다!

Reply:
소헌님, 추가적인 내용까지 제안해주셔서 감사합니다!
`Number()`를 사용하는 이유는 id가 숫자라는 것을 명확하게 표현하기 위한 목적이라는 것이 맞을까요?😃

`product.id.to === Number(id)`의 `to`라는 것이 궁금해 조사해보았는데요,
제가 못찾는 것인지...`to`를 검색해도 발견할 수가 없었고
무엇보다 `product.id.to === Number(id)`를 그대로 사용하니 코드가 원하는 대로 동작하지 않았습니다😢
혹시 `product.id.toString() === Number(id)`를 말씀하시는 것 일까요?

Reply:
엇 죄송해요 하영님 `to`는 삭제가 덜 된 오타입니닷

Reply:
앗 감사합니다!
`product.id`가 숫자이니 그에 맞춰서 `req.params.id`를 숫자로 만들어주는 형식이었군요!
제안해주신 방식으로 수정했습니다☺️

https://github.com/wecode-bootcamp-korea/nodejs-CRUD/pull/1/commits/5b4a54d1a63062e94b121b6212afcb3b1912cd59